packages/core/src/lib/wallet/wallet.types.ts

```diff
@@ -27,6 +27,7 @@ export interface Account {
 export interface SignInParams {
   contractId: string;
   methodNames?: Array<string>;
+  qrCodeModal?: boolean;
```

Comment:
The `qrCodeModal?` key is not needed in this interface since it is only a specific option for `Bridge Wallets` and the type is already declared [here](https://github.com/near/wallet-selector/blob/SQC-288/qr-code-in-ui/packages/core/src/lib/wallet/wallet.types.ts#L168-L170) and we make use of it in this [line](https://github.com/near/wallet-selector/blob/SQC-288/qr-code-in-ui/packages/core/src/lib/wallet/wallet.types.ts#L176)